portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,74 @@
+*{
+    margin: 0;
+    box-sizing: border-box;
+}
+body{
```

Comment:
"Use a space between the last selector and the declaration block." - https://google.github.io/styleguide/htmlcssguide.html#Declaration_Block_Separation

Here and elsewhere there's a `{` in this file.

Reply:
FYI, there's still some spaces missing  between the selector and the `{` in some places.